101-asp-net-app-service/azuredeploy.json

```diff
@@ -0,0 +1,134 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "name": {
+            "type": "string",
+            "metadata":{
+                "description": "That name is the name of our application. It has to be unique.Type a name followed by your resource group name. (<name>-<resourceGroupName>)"
+            }
+        },
+        "location": {
+            "type": "string",
+            "allowedValues":["Central US", "North Europe", "West Europe", "Southeast Asia", "East Asia", "West US", "East US", "Japan West", "Japan East", "East US 2", 

```

Comment:
remove allowedValues and set a defaultValue of [resourceGroup().location] - I know there are missing regions at the moment, but this list will create a maint problem